build.sh

```diff
@@ -35,43 +62,41 @@ build_type=RelWithDebInfo
 #extra_enable="-D HAVE_PARPORT=1"
 extra_enable=""
 
-build_flags=""
-
 case "${ostype}" in
     linux)
-	# try to find out whether this is an Embedded Linux
-	# platform (e.g. Raspberry Pi)
-	machine=$(uname -m)
-	if expr "${machine}" : '^\(arm\|aarch\)' >/dev/null
-	then
-	    extra_enable="${extra_enable} -D HAVE_LINUXGPIO=1 -D HAVE_LINUXSPI=1"
-	fi
-	;;
+        # try to find out whether this is an Embedded Linux
```

Comment:
We tend to leave the formatting/spacing of existing code alone. This so that we don't get unnecessarily distracted when reviewing code and/or trying to figure out which commits made changes to certain code parts.

Please revert to original spacing in this section.